### PR TITLE
FS-54 update swagger and dev docs for merchant report

### PR DESCRIPTION
**Pull request by jsmithbam**

### CODE REVIEW

File: source/docs/guides/merchant_report.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  toc: false
+  header_active: Guides
+---
+
+# Merchant Report
+
+## Report Overview
+
+The Merchant Report provides an overview of merchants progress through the boarding process. 
+You will be able to view information relating to a merchant or its submerchants including the account status, batch limits, cards eneabled and features.
```
Comment (david-light) [inline source/docs/guides/merchant_report.md]: enable spelt wrong

File: source/docs/guides/merchant_report.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+In addition, you will need use your Merchant ID and Reporting API Access
+Passcode from your Bambora account.  See the "Authentication" section below
+for more details on how to generate your access passcode to use in the request
+to the Merchant Report API.
+
+## API Requests
+
+### Request Headers
+
+Requests to the settlement endpoint should include two request headers:
```
Comment (david-light) [inline source/docs/guides/merchant_report.md]: This says settlement

File: source/docs/guides/merchant_report.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+// base 64 encode the two values concatenated with a colon
+yourAPIAccessPassCode = btoa(merchantID + ":" + reportingAPIPasscode);
+```
+
+In this example `yourAPIAccessPassCode` would be
+`MTIzNDU2Nzg5OmZkMzZkMjQ1YTM1ODQ0MzRiOTA0MDk2NTI1NTQ3ZjE3` which is the value
+used when making a call to the Merchant Report API.
+
+Alternatively, for convenience, you can also use the
+[form](https://dev.na.bambora.com/docs/forms/encode_api_passcode/) which does
```
Comment (david-light) [inline source/docs/guides/merchant_report.md]: should be '...which does the same calculation.' Missing a 'the'

File: source/docs/guides/merchant_report.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+API Passcode it is not an *encrypted* value, so this value should be treated with
+equal sensitivity/security as your Reporting API Passcode itself.
+
+### Parameters
+
+The Merchant Report can accept one parameter to control the data returned by
+the endpoint: `merchants`, which is supplied as part of the URL
+
+If not passed the data returned will be for all the submerchants of the Channel Partner requesting the report.
+If a submerchant is passed in then only data for that submerchant will be returned
```
Comment (david-light) [inline source/docs/guides/merchant_report.md]: I think submerchant should be sub-merchant to be consistent with other documentation

File: source/docs/guides/merchant_report.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+```shell
+curl --location --request GET 'https://api.na.bambora.com/v1/reports/merchants/{merchant_id}' \
+--header 'Authorization: Passcode <YOUR API ACCESS PASSCODE>' \
+--header 'Content-Type: application/json'
+```
+
+Replace `<YOUR API ACCESS PASSCODE>` with your API passcode.
+
+If you wish to only retrieve information for your own account then the request would be
```
Comment (david-light) [inline source/docs/guides/merchant_report.md]: This should be 
If you wish to retrieve information for all of your sub-merchants then the request would be:

**Review by david-light — CHANGES_REQUESTED** [review]

**Review by GolnazIrannejad-Bambora — APPROVED** [review]

**Review by david-light — APPROVED** [review]

**Pull request merged**